### DISCUSSION TITLE
How to get the execution status and result in real time

### DISCUSSION THREAD

**Discussion by Meet-for-your-drink** [StackStorm/st2#5336] [Q&A] [answered]
If I use st2 executiveAPI to submit an action implement interface, I can only have access to the id of the execution record. So if I want to know the execution result, I have to use this id to search the query interface. In the action interface in webui of st2, the execution status and the result output are updated in real time. In the console, there is neither the query request initiation nor the setup of a websocket connection. Therefore, I  want to know how this works？

**Comment by nzlosh** [comment, marked as answer]
Real-time output of actions is describe here https://docs.stackstorm.com/reference/action_output_streaming.html

**Comment by arm4b** [comment]
Just converted the Issue to Q/A Discussion.
